.travis.yml

```
@@ -1,20 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
 script: bash -ex .travis-docker.sh
 services:
   - docker
 sudo: false
 env:
  global:
-   - PACKAGE="publish"
+   - PACKAGE="opam-publish"
+   - OPAMSOLVERTIMEOUT=600
```

```suggestion
   - EXTRA_ENV="OPAMSOLVERTIMEOUT=600"
```
With the docker variants it works this way

---

Thanks! There is a solver timeout on most jobs... quite weird